docs/learning/workshop_a_precision_converter_fpga_integration_journey/index.rst

```diff
@@ -0,0 +1,693 @@
+
+**System Evaluation – Results comparison**
+
+=========================================== ========= ================================== ================================== ====================================
+Parameter                                   Datasheet Regular SPI controller             SPI Engine controller              SPI Engine controller
+                                                      (Fin=1kHz, SR=15KHz, Ain=-0.5dBFS) (Fin=1kHz, SR=15KHz, Ain=-0.5dBFS) (Fin=1kHz, SR=1.33MHz, Ain=-0.5dBFS)
+=========================================== ========= ================================== ================================== ====================================
+Signal-to-Noise SNR [dBFS]                     98.5                 14.81                               78.60                               77.70
+Spurious-Free Dynamic Range SFDR [dBFS]       112.5                 21.13                               92.97                               99.15
+Total Harmonic Distortion THD [dBFS]         -110.5                -45.65                              -99.20                                -110
+Signal-to-(Noise + Distortion) SINAD [dBFS]      98                 14.30                                  78                               77.20
+=========================================== ========= ================================== ================================== ====================================
+
+**System Evaluation**
+
+=========================================================== ====================== =====================
+Steps                                                       Regular SPI controller SPI Engine controller
+=========================================================== ====================== =====================
+Connect IIO Oscilloscope and visualize the captured signals           X                     X
+Run the Python script on Cora z7s / Cygwin                            X                     X
+Analyze results in VisualAnalog                                       X                     X
+Compare the results                                                   X                     X
+=========================================================== ====================== =====================
+
+Conclusions
+~~~~~~~~~~~
+
+1. A classic MCU can be used for converters that have the sampling rate up to 100kSPS.
+2. Maximum performance, in terms of sampling rate, SNR, THD can be only achieved with an FPGA.
+3. SPI Engine is a highly flexible and powerful open-source SPI controller
+   framework which can interface a wide range of precision converters.
+4. We are now familiar with the COS group open-source solution stack.
+
+**Thank You!**
+
+**Related Presentations**
+
+- My customer uses a FPGA in his product. Now what?
+- ADALM2000 in real life applications
+- Just enough Software and HDL for High-Speed designs
+- Hardware and Software Tools for Precision Wideband Instrumentation
```

Comment:
Maybe we could link to those, no?

Reply:
Cannot find this FTC presentations on System Level Docs, can you provide some hints where should I take the links?

Talking to Paul he is not aware that this presentations were ported...